src/components/DataViewFilterDropdownButtons/DataViewFilterDropdownButtons.styled.tsx

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
+import theme from "@root/theme";
+import Button from "../Button";
+import ButtonRow from "../ButtonRow";
+
+export const StyledWrapper = styled.div`
+  & {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 1rem;
+  }
+`;
+
+export const StyledButton = styled(Button)`
+  &.normalButton > button {
+    font-size: 16px;
```

Comment:
I know the ticket says 16px, but let's keep it as 14... this is something that we've already talked with the design team but they haven't changed their specs.